.pre-commit-config.yaml

```diff
@@ -0,0 +1,10 @@
+files: '.md$'
```

Comment:
need to figure out and check if it can work properly with the existing hooks

Reply:
you're right - there's a conflict. Removed in favour of husky now,